pkgs/top-level/all-packages.nix

```diff
@@ -41544,4 +41544,5 @@ with pkgs;
 
   wttrbar = callPackage ../applications/misc/wttrbar { };
 
+  espup = callPackage ../development/tools/espup { };
```

Comment:
This doesn't seem to be at the right place, please sort it out.

Reply:
Sorry, I didn't see categories prefixed with `###`. Fixed.